File: ProvaGFT/src/exercicio01/Gerente.java
package exercicio01;

public class Gerente extends Funcionario {

	public Gerente(String nome, int idade, double salario) {
		super(nome, idade, salario);
		
	}
	
	@Override
	public void Bonificacao() {
		this.setSalario(this.getSalario()+10000.00);	
		System.out.println(this.getNome() + ", com o salário de " + this.getSalario());
	}
	

}

File: Prova/Gerente.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Prova
{
    class Gerente : Funcionarios
    {
        public Gerente(string nome, int idade, double salario) : base(nome, idade, salario)
        {
        }
        public void Bonificacao()
        {
            Salario = Salario + 10000.00;
            Console.WriteLine("O salário da " + Nome + " é de " + Salario + " reais.");
        }
    }
}
